Dockerfile

```diff
@@ -11,14 +11,19 @@ RUN apt-get update -yqq && apt-get install -yqq --no-install-recommends \
 # sdrN for volumes and symlinks
 RUN bash -c 'for i in $(seq 1 24); do ln -s /sdr/$i /sdr$i; done'
 
+WORKDIR /usr/src/app
+ENV BUNDLE_PATH /gems
+ENV RUBYLIB /usr/src/app/lib
+
+FROM development AS production
```

Comment:
This was my only WTF? moment reading the diff. I think this deserves a comment because I have to look up to line 1 and maybe would have expected something like `base` there (not that I prefer "base", it's just the nomenclature I have come to expect) instead of `development`.